tests/cdp.spec.ts

```diff
@@ -54,3 +54,21 @@ test('cdp server reuse tab', async ({ cdpEndpoint, startClient }) => {
 \`\`\`
 `);
 });
+
+test('should throw connection error and allow re-connecting', async ({ cdpEndpoint, startClient }) => {
+  const port = 3200 + (+process.env.TEST_PARALLEL_INDEX!);
```

Comment:
I know it's existed before, but what the heck is this env var? Shouldn't it be `testInfo.parallelIndex`?

Reply:
Yes.